name: "env1"
robot_radius: 0.345
goal_tolerance: 0.5

start: [-10.0, -10.0]
goal: [0.0, 0.0]

# List of rectangular obstacles defined as [cx, cy, width, height]
obstacles: [
 [-10.,  -8.,  1.0,  1.0],
 [ -6.,  -8.,  1.0,  1.0],
 [ -5.,  -6.,  1.0,  1.0],
 [ -5.,  -5.,  1.0,  1.0],
 [ -5.,  -4.,  1.0,  1.0],
 [ -5.,  -1.,  1.0,  1.0],
 [ -2.,  -1.,  1.0,  1.0],
 [ -3.,  -1.,  1.0,  1.0],
 [ -2.,   0.,  1.0,  1.0],
 [ -1.,   1.,  1.0,  1.0],
 [  2.,   3.,  1.0,  1.0],
 [  2.,   2.,  1.0,  1.0],
 [  5.,   5.,  1.0,  1.0],
 [  3.,   3.,  1.0,  1.0]
]